dashboard/README.md

```diff
@@ -28,6 +28,28 @@ Protocol level data involves subgraph data that does not apply to any specific p
 
 Below the table, timeseries data is divided into their own sections. On the Protocol level the timeseries entities are `financialsDailySnapshots`, `usageMetricsDailySnapshots`, and depending on the subgraph schema version possibly `usageMetricsHourlySnapshots`. All of the fields on these entities are mapped through and rendered into charts plotting each data point on a given snapshot.
 
+#### Data Issues Auto Detected for QA purposes:
+
+- Any field has a cumulative timeseries value of 0. Varying severity levels depending on field.
+- No snapshots for a protocol
+- Cumulative field lowers from any given snapshot to the next
+- TVL on protocol is less than 1000 or above 1000000000000
+- Total revenue is not equal to protocol side + supply side (with tolerance)
+- Individual transactions do not add up to total transaction number
+- Protocol level current values are falsey
```

Comment:
typo i think you meant `false` instead

Reply:
I mispelled `falsy`, meaning a value like null, undefined, false, "", 0 NaN, etc